### DISCUSSION TITLE
[Question] Obfuscation by default

### DISCUSSION THREAD

**Discussion by pollosp** [open-telemetry/opentelemetry-ruby-contrib#442] [Q&A] [answered]
Hello, 
I was checking instrumentation for MySQL and Mongo and looks there is no consistency into the obfuscation parameters by default. 

Mysql doesn't obfuscate and Mogodb does. 

Is there any reason behind this? 

Usually I prefer obfuscation by default as sensitive data could end up in a way you don't want easily

Thanks

**Comment by arielvalentin** [comment, marked as answer]
@pollosp Behaviors and default sanitization of `db.statement` attributes is under discussion here: https://github.com/open-telemetry/opentelemetry-specification/issues/3150

Changes to the specification requiring sanitization by default was merged only 2 weeks ago so it looks like we are behind: https://github.com/open-telemetry/opentelemetry-specification/pull/3127

**Comment by cheempz** [comment]
@arielvalentin there is a general SQL improvement issue https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/32, wondering if setting obfuscate by default would be covered as part of that, or if we can start with simple individual PRs to just toggle the default for PG, MySQL2 and Dalli instrumentation (which we'd be happy to help on!).
